scala-ts-mode.el

```diff
@@ -445,6 +445,30 @@
          (back-to-indentation)
          (point))))))
 
+(defun scala-ts--find-parent (node pred)
+  "Find first parent node of NODE for which PRED return t."
+  (let ((result nil))
+    (cl-loop for cursor = node
+             then (treesit-node-parent cursor)
+             while (and (not result) cursor)
+             if (funcall pred cursor)
+             do (setq result cursor))
+    result))
```

Comment:
I'm not sure what the difference between `treesit-parent-until`, which is built-in to the `treesit.c` module and this function is? Could you please elaborate?

Reply:
I didn't notice existence of `treesit-parent-until`, I'm using it now instead of `scala-ts--find-parent`. Thanks.